crates/common/src/version.rs

```diff
@@ -0,0 +1,11 @@
+// The version of the Foundry CLI.
+pub const VERSION_MESSAGE: &str = env!("FOUNDRY_SHORT_VERSION");
+
+// Whether the version is a nightly build.
+pub const IS_NIGHTLY_VERSION: &str = env!("FOUNDRY_IS_NIGHTLY_VERSION");
```

Comment:
```suggestion
pub const IS_NIGHTLY_VERSION: bool = env!("FOUNDRY_IS_NIGHTLY_VERSION") == "true";
```

Alternatively, `option_env!().is_some()` with a conditional in build.rs as well